### ISSUE TITLE
Unable to access blocked site after disconnected 3rd party VPN

### ISSUE THREAD

**Issue by fffw** [getlantern/lantern#3506]
When VPN connected, blocked sites become accessible, so Lantern adds them to the bypass list in PAC file.

It only happen in that specific scenario and one can always restart Lantern to solve it, but it's better to recover without user intervention.

One way should be to detect network change in Go code and reset bypass list.

**Comment by myleshorton** [comment]
A related issue is that sometimes we get other false positives where some early requests succeed for whatever reason but later requests don't.

**Comment by fffw** [comment]
Yeah it's correlated with https://github.com/getlantern/lantern/issues/4194

**Comment by fffw** [comment]
We can simply invalidate the bypass domains list when any network changes are detected. [Windows](https://msdn.microsoft.com/en-us/library/windows/desktop/aa366334(v=vs.85).aspx), [Mac](http://stackoverflow.com/questions/1083701/how-to-check-for-an-active-internet-connection-on-ios-or-osx).
